tests/test_file_readers.py

```diff
@@ -1,14 +1,21 @@
 import pyarrow as pa
```

Comment:
I would change the name of this test to be more explicit what it is testing and add a comment about what the test is testing.

From looking at it I think it is testing whether we get the same dataframe whether or not we pass a schema parameter.

We might want to add some tests that look at the behaviour is a non csv or json file is passed as well.